tsconfig.json

```diff
@@ -17,7 +17,8 @@
     "paths": {
       "core-js/es7/reflect": ["node_modules/core-js/proposals/reflect-metadata"],
       "core-js/es6/": ["node_modules/core-js/es/"]
-    }
+    },
+    "skipLibCheck": true
```

Comment:
Why skip library check?

Reply:
Hey @gycgabriel, thanks for catching this. I used this configuration to bypass the following compilation error from TypeScript:

![error](https://user-images.githubusercontent.com/34131671/182742166-9451df17-7e08-456d-99e6-66980b72a65c.png)

After some googling, I think this is due to the version of TypeScript being old, which causes it to throw an error when compiling the files in the module `@github/markdown-toolbar-element`.
- our current TypeScript version is `3.5.3`
  - users have reported that this issue is resolved when they upgrade the TypeScript version to `3.7.4`
    - [link to stackoverflow post ](https://stackoverflow.com/questions/60092642/ts1086-an-accessor-cannot-be-declared-in-ambient-context)
- a PR has been submitted by @kevinchua6 in #963 
  - updates TypeScript to version `4.0.8`
  - which should resolve the issue

I think the safest option would be for me to remove this configuration option, and then to wait for #963 to be approved and merged first before seeing if it resolves the compilation error? Since there could be some unforeseen consequence in the future caused by including this.